### PR TITLE
Added Dot2Ct & Dot-Bracket format + implementation in drawrnajs

### PR DESCRIPTION
**Pull request by yhoogstrate**
I added the tool Dot2Ct from the RNAStructure package, including a datatype definition for Dot-Bracket formatted files. The Dot-Bracket format is now also implemented in drawrnajs.

### CODE REVIEW

File: tools/rna_tools/dot2ct/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+categories:
+- Sequence Analysis
+- RNA
+description: "Dot-Bracket to Connect Table (CT)"
+long_description: |
+  Converts the first sequence in a Dot-Bracket file into a CT file
+
+  Party of the RNAStructure package.
```
Comment (bgruening) [inline tools/rna_tools/dot2ct/.shed.yml]: s/Party/Part/

File: tools/rna_tools/dot2ct/datatypes_conf.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<datatypes>
```
Comment (bgruening) [inline tools/rna_tools/dot2ct/datatypes_conf.xml]: @yhoogstrate can we stick to 4spaces intent for all files? This is for sure debatable, but it is kind of a standard.

File: tools/rna_tools/dot2ct/datatypes_conf.xml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+	<datatype_files>
+		<datatype_file name="RNAStructure.py"/>
+	</datatype_files>
```
Comment (bgruening) [inline tools/rna_tools/dot2ct/datatypes_conf.xml]: Awesome, we will have today a datatype discussion: https://github.com/galaxy-iuc/standards/issues/13
So we will know soon if we can move this straight into Galaxy main.

File: tools/rna_tools/dot2ct/datatypes_conf.xml (hunk 2 above)
Comment (bgruening) [inline tools/rna_tools/dot2ct/datatypes_conf.xml]: For more standards please refer to: http://galaxy-iuc-standards.readthedocs.org/en/latest/

File: tools/rna_tools/dot2ct/lib/galaxy/datatypes/RNAStructure.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+import galaxy
+import galaxy.model
+import galaxy.datatypes
+import galaxy.datatypes.data
+
+from galaxy.datatypes.data import Text
+from galaxy.datatypes.metadata import MetadataElement
+
+from galaxy.datatypes import metadata
+from galaxy.datatypes import data
```
Comment (bgruening) [inline tools/rna_tools/dot2ct/lib/galaxy/datatypes/RNAStructure.py]: I think most of the import are not needed, isn't it?

**Comment by yhoogstrate** [comment]
Thanks for the feedback. I have addressed the comments above, and I managed to get the visualization button working for the custom DotBracket data type.

**Comment by bgruening** [comment]
@yhoogstrate thanks a lot! Will upload it shortly ...

**Pull request merged**